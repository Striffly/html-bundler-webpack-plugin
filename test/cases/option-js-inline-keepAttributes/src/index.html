<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <script src="./app1.js" id="app1" type="text/javascript"></script>
  <script src="./app2.js" defer="defer"></script>
  <script src="./app3.js" defer></script>
  <script src="./app4.js" data-id="remove me" async></script>
</head>
<body>
  <h1>Test</h1>
</body>
</html>