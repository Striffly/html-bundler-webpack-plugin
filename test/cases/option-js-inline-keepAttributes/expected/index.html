<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <script id="app1" type="text/javascript">console.log("keep attributes: id, type");</script>
  <script defer="defer">console.log('keep attribute: defer="defer"');</script>
  <script defer>console.log("keep attribute: type: defer");</script>
  <script>console.log("remove all attributes");</script>
</head>
<body>
  <h1>Test</h1>
</body>
</html>